LongWarOfTheChosen/Config/XComGameData_WeaponData.ini

```diff
@@ -959,6 +959,49 @@ LMG_ALL_RANGE[24] = -20
 LMG_ALL_RANGE[25] = -25
 LMG_ALL_RANGE[26] = -30
 
+MID_LONG_ALL_RANGE[0] = -10
+MID_LONG_ALL_RANGE[1] = -10
+MID_LONG_ALL_RANGE[2] = -7
+MID_LONG_ALL_RANGE[3] = -3
+MID_LONG_ALL_RANGE[4] = 0
+MID_LONG_ALL_RANGE[5] = 0
+MID_LONG_ALL_RANGE[6] = 0
+MID_LONG_ALL_RANGE[7] = 0
+MID_LONG_ALL_RANGE[8] = 0
+MID_LONG_ALL_RANGE[9] = 0
+MID_LONG_ALL_RANGE[10] = 0
+MID_LONG_ALL_RANGE[11] = 0
+MID_LONG_ALL_RANGE[12] = 0
+MID_LONG_ALL_RANGE[13] = 0
+MID_LONG_ALL_RANGE[14] = 0
+MID_LONG_ALL_RANGE[15] = 0
+MID_LONG_ALL_RANGE[16] = 0
+MID_LONG_ALL_RANGE[17] = 0
+MID_LONG_ALL_RANGE[18] = 0
+MID_LONG_ALL_RANGE[19] = 0
+MID_LONG_ALL_RANGE[20] = 0
+MID_LONG_ALL_RANGE[21] = 0
+MID_LONG_ALL_RANGE[22] = -10
```

Comment:
Now that i look at it, -10 per tile does sound extreme, I'd probably drop it to -5 or -3 aim per tile.

Reply:
OK, -4 is it 😆